python/examples/llama-index/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This run script is used to run the backend and frontend apps in the background by docker
+
+# Run the backend app from the /backend in the background
+cd backend && python main.py &
+
+# Run the frontend app from the /frontend in the background
+cd frontend && npm run dev &
+
+pid=
+trap 'echo SIGINT; [[ $pid ]] && kill $pid; exit' SIGINT
+trap 'echo SIGTERM; [[ $pid ]] && kill $pid; exit' SIGTERM
+echo Starting script
+sleep 10000 & pid=$!
+wait
+pid=
```

Comment:
Do we need this file with the current Docker configuration? This looks like it would be used to run the frontend and backend in the same container, but we're not doing that.

Reply:
Yeah please delete. Not needed

Reply:
This is the auto-generated initialization script, I'm thinking maybe we should keep it in case a user wants to run this locally?

Reply:
@anticorrelator its not I added it 😅

Reply:
oh! ok 😅